src/commands/index.js

```diff
@@ -87,7 +87,7 @@ const roll = {
         const diceValue = diceNumbers[1];
         const { total, details } = Array.from(new Array(diceNumber)).reduce(
           (acc) => {
-            const value = Math.round(diceValue * Math.random()) + 1;
+            const value = Math.round((diceValue - 1) * Math.random()) + 1;
```

Comment:
We can have -1 here if someone tries to roll 0d0 . Change the condition line 81-82 also

Reply:
> someone tries to roll 0d0

The condition lines 78-83 is filtering the 0d0